Dockerfile

```
@@ -0,0 +1,34 @@
+# 1: Build the exe
+FROM rust:1.56 AS builder
+
+# WORKDIR /usr/src
+
+# RUN USER=root cargo new agora
+
+# COPY Cargo.toml Cargo.lock /usr/src/agora/
+
+WORKDIR /usr/src/agora
+
+COPY . ./
+RUN cargo build --release
+
+FROM ubuntu AS final
+
+ARG APP=/usr/src/app
+ENV APP_USER=agora
+
+RUN useradd --system ${APP_USER}
+
+COPY --from=builder /usr/src/agora/target/release/agora ${APP}/bin/agora
```

Can we put this in `/usr/local/bin/agora`, so that it can be called by just running `agora`?

```suggestion
COPY --from=builder /usr/src/agora/target/release/agora /usr/local/bin/agora
```